doc/man5/config.pod

```diff
@@ -486,6 +487,28 @@ built-in entropy sources will be used.
 
 =back
 
+=head2 ASN.1 String‑Table Configuration
+
+The name B<stbl_section> in the initialization section names the section
+containing overrides for the built‑in ASN.1 string‑table size limits
+(defined in C<tbl_standard.h>). Within this section, each entry is a
+name/value pair where the B<name> is an ASN.1 object short or long name,
+and the B<value> is a comma‑separated list of parameters in the form
+C<min:<n>>, C<max:<n>>, C<mask:<mask>>, or C<flags:<flags>>.
```

Comment:
To repeat my comment from #27420, I think it would be useful to describe the semantics of the `mask` and `flags` fields.